speed-dreams/source-2.2.3/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6.0 FATAL_ERROR)
 
 MESSAGE(STATUS CMAKE_VERSION = "${CMAKE_VERSION}")
+MESSAGE(STATUS "CMAKE_CXX_COMPILER_LAUNCHER = ${CMAKE_CXX_COMPILER_LAUNCHER}")
```

Comment:
Is this message still needed? Or does it also hold valueable info for when its building on gh?

Reply:
I was just looking and one of my commits was not pushed, while this could be useful information for debugging I'll remove it